src/app/billingentity/billing-entity.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="flex flex-row flex-wrap justify-content-between mb-2 gap-2">
+  <h1 class="pt-0 mt-0 mb-0" i18n id="billingentities-title">Billing Entities</h1>
+</div>
+
+<ng-container *ngIf="(billingEntityService.loading$ | ngrxPush) === false">
+  <ng-container *ngFor="let billingEntity of billingEntities$| ngrxPush">
+    <div class="surface-card p-4 shadow-2 border-round mb-4">
+      <div class="flex flex-row justify-content-between">
+        <div class="text-3xl font-medium text-900 mb-3">
+          {{ billingEntity.metadata.name }}
+        </div>
+        <div>
+          <a
+            [routerLink]="[billingEntity.metadata.name]"
+            class="text-blue-500 hover:text-primary cursor-pointer mr-2"
+            i18n-title
+            title="Edit members">
```

Comment:
```suggestion
            title="Edit details">
```

Technically, it is only view for now. It will be edit later so just use edit, and we do not have to change later on.